store/member.go

```diff
@@ -63,6 +63,37 @@ func (s *Store) CreateMember(ctx context.Context, create *api.MemberCreate) (*ap
 	return member, nil
 }
 
+// GetDefaultAssigneeIDFromWorkspaceOwnerOrDBA finds a default assignee from the workspace owners or DBAs.
+func (s *Store) GetDefaultAssigneeIDFromWorkspaceOwnerOrDBA(ctx context.Context) (int, error) {
```

Comment:
Can we use https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/store/member.go?L67 instead?